src/irmin-test/store.ml

```diff
@@ -772,7 +772,7 @@ module Make (S : S) = struct
             watch (n - 1)
       in
       let v1 = "X1" in
-      let v2 = "X2" in
+      let _v2 = "X2" in
```

Comment:
This one's a bit odd; shouldn't it have been a warning if it wasn't being used? (In which case, we should just be able to remove it outright.)

Reply:
Good catch, thank you!